aws/eks.ts

```diff
@@ -364,15 +323,15 @@ export class AwsCloud implements model.Cloud {
     const clusterInstances: aws.rds.ClusterInstance[] = [];
     const rdsInstanceName = util.name("rds-inst");
 
-    for (const range = {value: 0}; range.value < 2; range.value++) {
-      clusterInstances.push(new aws.rds.ClusterInstance(`${rdsInstanceName}-${range.value}`, {
-        identifier: `${rdsInstanceName}-${range.value}`,
+    for(let index = 0; index < 2; index++) {
```

Comment:
I didn't get why the `for` loop was using an object before, but this is simpler and works.